beacon-chain/sync/initial-sync/service.go

```diff
@@ -111,12 +111,16 @@ func (s *Service) Start() {
 	}
 
 	if genesis.After(roughtime.Now()) {
-		log.WithField(
-			"genesis time", genesis,
-		).Warn("Genesis time is in the future - waiting to start sync...")
-		time.Sleep(roughtime.Until(genesis))
```

Comment:
This was not needed - it was preventing validators from getting duties before chainstart if a genesis time exists

Reply:
Great change!